index.html

```diff
@@ -8,44 +8,61 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <div class="div-container">
-      <section>
-        <header class="header">
+        <header>
           <div class="My-Logo">
-            <h1>My Logo</h1>
-          </div>
-          <div class="header-Line-parent">
-            <div class="header-Line-1"></div>
-            <div class="header-Line-2"></div>
-            <div class="header-Line-3"></div>
+            <a href="#"><h1>My Logo</h1></a>
           </div>
+          <nav class="header-Line-parent">
+            <img src="Union.svg" alt="menu-icon">
+          </nav>
         </header>
-      </section>
-
-      <section class="section-1">
-        <div class="heading-1">
-          <h1 class="First-heading">
-            I'm Mariane <br />
-            Glad to see <br />
-            you!
-          </h1>
-          <span class="First-heading-description">
-            I'm a software developer! I can help you build a prduct,feature or
-            website Look through some of my work and experience! if you like
-            what you see and have a project you need coded, don't hesitate to
-            contact me.
-          </span>
-          <h3 class="Connect-heading">LET'S CONNECT</h3>
-        </div>
-      </section>
 
-      <div class="icons-div">
-        <img class="twitter icons" src="Vector (5).svg" alt="" />
-        <img class="linkedin icons" src="Linkedin.svg" alt="" />
-        <img class="medium icons" src="Vector (2).svg" alt="" />
-        <img class="github icons" src="Vector.svg" alt="" />
-        <img class="angellist icons" src="Vector (1).svg" alt="" />
-      </div>
-    </div>
+      <main>
+        <section class="section-1">
+          <img class="background-img" src="header-shapes mobile@2x.svg" alt="">
+          <div class="heading-1">
+            <h1 class="First-heading">
+              I'm Hasnat <br />
+              Glad to see <br />
+              you!
+            </h1>
+            <p class="First-heading-description">
+              I'm a software developer! I can help you build a prduct,feature or
+              website Look through some of my work and experience! if you like
+              what you see and have a project you need coded, don't hesitate to
+              contact me.
+            </p>
+            <h3 class="Connect-heading">LET'S CONNECT</h3>
+          </div>
```

Comment:
Good job @hasnatali1947 😄, but make sure you update the styling of your headline section to better match the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=vByip9LCrbQjAJIk-0) as is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.). Kindly fix some font-size, margin, font weigh and color 👍

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | | <img width="226" alt="mobile 2 0" src="https://user-images.githubusercontent.com/57408419/219616755-cf131229-f222-4666-8d7e-1849ec48f9be.png"> | |![mobile](https://user-images.githubusercontent.com/57408419/219652618-c9f30922-95b4-4671-b3f2-f890f012c09d.jpg) | |